examples/otlp/docker/otel-agent-config.yaml

```diff
@@ -0,0 +1,38 @@
+receivers:
+  opencensus:
+  zipkin:
+    endpoint: :9411
+  jaeger:
+    protocols:
+      thrift_http:
+
+
+exporters:
+  opencensus:
```

Comment:
Since this is the java repo and the example is named otlp, should we just use otlp between the two? Or do we even need to run two collectors for this demo?

Reply:
yeah, I think we can remove the agent for this demo. I'll pull it out.